include/bluetooth/mesh/sensor_cli.h

```diff
@@ -602,7 +602,6 @@ int bt_mesh_sensor_cli_series_entry_get(
  *                         has been changed to the number of columns in the
  *                         response.
  *  @retval -ENOTSUP       The sensor doesn't support series data.
- *  @retval -ENODEV        The sensor server doesn't have the given sensor.
```

Comment:
Suggest changing this to `-ENOENT` instead of removing it

Reply:
Oh, I did not just remove it for no reason, the **exact** same line with the **exact** same content also exists a few lines above.

Reply:
Oh, I see, my bad